README.txt

```diff
@@ -24,17 +27,15 @@ Benefits
 Users browse, search, and submit entries into the knowledge base. Entries can disprove, support, replicate, or peer-review other entries. The website calculates and informs users the status of an entry's validity using this system. An entry is most meritable if it follows the scientific method correctly. An entry is least meritable if it is just an opinion, or comment, and is less visible when sorting. Lower merit entries should not be dismissed, however, because they often inspire more meritable entries to be submitted.
 
 
-Mission Statement
+Mission Statement:
+To teach and engage 7+ billion people in investigative science and critical thinking. To maximize collaboration and assist humanity in reaching its collective intellectual potential.
 
-To engage everyone in problem-solving, using the scientific method as the most meritable method. To ensure that, through Methodocracy, humanity always has the ability to problem solve as much and as fast as possible.
```

Comment:
This stays for now for the following reasons; it needs to say "scientific method" in order for the IRS to approve our nonprofit status, the second sentence also serves to make sure that all requirements need to be met in order for problem solving to take place, such as our survival and our quality of life high enough to allow us to spend time problem solving on other things beside the lower tiers of maslow's hierarchy of needs.